src/compute/src/render/join/delta_join.rs

```diff
@@ -437,7 +345,7 @@ where
                 prev_key
                     .iter()
                     .map(|e| e.eval(&datums_local, &temp_storage)),
-                updates_key_types.as_deref(),
+                None,
             )?;
```

Comment:
I think we can inline the `try_from_datum_iter` call, but also fine to do it in a follow-up PR.

Reply:
Yes I also saw it, too late.